### PR TITLE
Code

### PR DESCRIPTION
**Pull request by ABohynDOE**
I've finished the `csv2json.py`script that converts a csv file to a JSON script and I've added some more documentation

### CODE REVIEW

File: docs/glossary.md
Comment (dmbarreiro) [inline docs/glossary.md:7]: github's markdown does not understand late syntax, that's a pity. There are some solutions out there but nothing we can directly use. We can talk about it tomorrow but for the moment I find this OK

**Review by dmbarreiro — CHANGES_REQUESTED** [review]

File: utils/csv2json.py
Comment (ABohynDOE) [reply utils/csv2json.py:1]: Good idea

**Review by dmbarreiro — APPROVED** [review]
All good for me, let's wait for @saif-analyze or @mariobecerra  review of csv2json.py
Good job!

**Comment by dmbarreiro** [comment]
I will merge this already, if something needs to be adjusted we will do it later

**Pull request merged**
